config/config.go

```diff
@@ -40,6 +41,7 @@ type Config struct {
 	MaxRowsReturned              int           `envconfig:"MAX_ROWS_RETURNED"`
 	Mongo                        mongo.MongoDriverConfig
 	KafkaConfig                  KafkaConfig
+	EnableURLRewriting           bool
```

Comment:
This should have an associated envconfig value like the others so it can be set in dp-configs